shoop/admin/modules/products/forms/package_forms.py

```diff
@@ -21,10 +23,14 @@
 from .parent_forms import ProductChildBaseFormSet
 
 
+class PackageProductChoiceWidget(ProductChoiceWidget):
```

Comment:
This should probably be in ``shoop.admin.forms.widgets`` with all the other widgets.